core/src/scheduler/resource/Resource.cpp

```diff
@@ -184,7 +184,7 @@ Resource::loader_function() {
 
 void
 Resource::executor_function() {
-    SetThreadName("taskexector_th");
+    SetThreadName("task_executor_th");
```

Comment:
TOO LONG.

Reply:
use "taskexecutor_th" now, it's 15 charactors